### DISCUSSION TITLE
Using mongodb atlas

### DISCUSSION THREAD

**Discussion by bulldoguk** [apostrophecms/apostrophe#3386] [Q&A]
I'm trying to fire up an APOS3 instance using Mongodb Atlas, which worked great on my APOS2 instance, but using the same data/local.js file does not seem to work on v3.  When I try to setup my admin user, I get an error referencing the DB host as 127.0.0.1 - where is this setting and how can I override it??

**Comment by boutell** [comment]
Hi @bulldoguk,

There are two good ways.

One is to set the `APOS_MONGODB_URI` environment variable. This is usually more convenient than `data/local.js` in cloud environments.

Or you can still do it the old way. You just have to remember that the module name has changed:

```javascript
module.exports = {
  modules: {
    '@apostrophecms/db': {
      uri: 'your URI here'
    }
  }
};
```

**Reply by JuanIrache** [reply]
I might be missing something very obvious, as I just started using Apostrophe today, but I get this error

> @apostrophecms/db (project level): uri is not a valid top level property for an Apostrophe 3.x module. Make sure you nest regular module options in the new "options" property.

Edit: Nevermind. I nested URI within an options object and all seems to be good now

**Comment by boutell** [comment]
Hi Juan, did you try doing that? "uri" needs to be nested in an "options"
object, as do all module options (as opposed to module sections, like
"methods" or "init" or "fields").

On Thu, Apr 7, 2022 at 8:08 AM Juan Irache ***@***.***> wrote:

> I might be missing something very obvious, as I just started using
> Apostrophe today, but I get this error
>
> @apostrophecms/db (project level): uri is not a valid top level property
> for an Apostrophe 3.x module. Make sure you nest regular module options in
> the new "options" property.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/apostrophecms/apostrophe/discussions/3386#discussioncomment-2524567>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAH27IKMNJK7IESBBXYN3LVD325RANCNFSM5DKT2PVA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***
> com>
>


-- 

THOMAS BOUTELL | CHIEF TECHNOLOGY OFFICER
APOSTROPHECMS | apostrophecms.com | he/him/his

**Reply by JuanIrache** [reply]
Yes, sorry. I edited my question when I made it work. Initially that didn't seem to be working, but I had a mistake on my URI. All good on this front
